.github/workflows/ci.yml

```diff
@@ -78,13 +78,13 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+      - run: sudo apt-get update
       - name: Setup OCaml
         uses: ocaml/setup-ocaml@v2
         with:
           ocaml-compiler: 4.13.x
```

Comment:
Can be perfectly done in an other PR but reviewing this makes me realize that `4.14`should be the "default" version used to build the js app and under MacOS/Windows...